java/rocksjni/comparatorjnicallback.cc

```diff
@@ -14,7 +14,10 @@ namespace ROCKSDB_NAMESPACE {
 ComparatorJniCallback::ComparatorJniCallback(
     JNIEnv* env, jobject jcomparator,
     const ComparatorJniCallbackOptions* options)
-    : JniCallback(env, jcomparator), m_options(options) {
+    : JniCallback(env, jcomparator),
+      m_options(std::make_unique<ComparatorJniCallbackOptions>(
```

Comment:
Can't we use default copy ctor (`make_unique<ComparatorJniCallbackOptions>(*options)`) and new constructor not needed?